contracts/README.md

```diff
@@ -0,0 +1,180 @@
+# Devnet contracts for development
+
+This folder contains Cairo and Solidity contracts for developement of Devnet.
+If you wish you to check specifically one of the two chains contracts, please check:
+1. `solidity` folder for Ethereum related contracts.
+2. `cairo` folder for Starknet related contracts, and example of how to work with starknet without
+   running a L1 node.
+
```

Comment:
note for devx:
we should note here somewhere the existence of readme.md files in both /cairo and /solidity , because there are instructions on installing Scarb etc.
Also to list all dependencies (like starkli), links to them etc.

Reply:
Good point, the updated README is here: https://github.com/glihm/starknet-devnet-rs/tree/7dd5d6e0fe8f14aca40ebd562cc3c5d5e16ce873/contracts/l1-l2-messaging with now an explicit reference to other READMEs.

Reply:
The main `README.md` in the repo root should link to the one in `contracts/l1-l2-messaging`. E.g. by introducing a new `Development` subsection. Similar to e.g. `## Development - Testing`, there could be `## Development - L1-L2 (postman)`

Reply:
Fixed in 483c52eefd1fb421a1f9396620c2d1a30fadd842 (and typo corrected in c00a45ddc5b47174afb65a2c9587b37d06294de7).